microcosm_sqlite/migrate.py

```diff
@@ -0,0 +1,252 @@
+"""
+Alembic CLI with simplified configuration.
+
+Alembic is a terrific tool that makes some unfortunate choices about configuration,
+expecting a verbose directory structure with several layers of configuration.
+
+This module monkey patches Alembic's CLI tool to work better within a microcosm.
+
+To use this entry enty point instead of the Alembic CLI:
+
+ 0. Don't use `alembic init`
+
+ 1. Define a `Base` like
+
+        Base = DataSet.create("foo")
+
+ 2. Add a `sqlite-migrations/foo` directory within your source tree.
+
+    This directory does not need to be an importable Python module, but it should
+    be included as part of your distribution so that migrations ship with the service.
+
+ 3. Initialize your object graph (including your models):
+
+        from microcosm.api import create_object_graph
+
+        graph = create_object_graph(
+            name="example",
+            loader=load_from_dict(
+                sqlite=dict(
+                    use_foreign_keys="False",
+                )
+            )
+        )
+
+    The migrations directory is loaded by default assuming that the `name` attribute
```

Comment:
With points 2 and 3 it's not entirely clear to me what the migrations directory should be named. Is `sql-migrations/foo` above the directory with the migrations?